src/main/java/com/jlr/ttl/poc/loadmanager/models/Location.java

```diff
@@ -5,24 +5,65 @@
 import jakarta.persistence.Id;
 import jakarta.persistence.Table;
 
+import java.util.Date;
+
 @Entity
 @Table(name = "locations")
 public class Location {
     @Id
+    @Column(name = "id")
     private String loc_code;
 
+    @Column
+    private String description;
+
+    @Column(name = "end_date")
+    private Date date;
+
     public Location() {
```

Comment:
Remove constructors and getters and setters. Use the following annotations instead:

AllArgsConstructor
NoArgsConstrutctor
Data
Build